generators/gen_reactnative.py

```diff
@@ -117,6 +119,29 @@ def generate(self, definitions: list):
 
     def _generate_ts_index(self, definitions: list):
         output_path = prepare_output_path(self.output_folder, OUTPUT_TS_FOLDER, OUTPUT_TS_INDEX)
+        predicate = lambda definition: (definition['name'] not in ["DdSdk", "DdSdkConfiguration"])
```

Comment:
I guess at least we could extract those hardcoded values into some static constants or maybe just create a static set : `NOT_EXPOSED_DEFINITIONS`

Reply:
thanks! I also added a test to check if exclusion works correctly.